monitor/cmd/monitor-server/server.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/mesosphere/performance/api/bigquery"
```

Comment:
right, it is untied from bigquery
the only reason i import this library is to get a structure, i can easily replace it.